code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -42,6 +42,9 @@
 		tally += (283.222 - bodytemperature) / 10 * 1.75
 	tally += stance_damage // missing/damaged legs or augs affect speed
 
+	if(slowdown)
+		tally *= 2
```

Comment:
You shouldn't double this since slowdown can go negative, adding 1 should be enough
```suggestion
		tally += 1
```